nx.json

```diff
@@ -0,0 +1,29 @@
+{
+    "extends": "nx/presets/npm.json",
+    "tasksRunnerOptions": {
+        "default": {
+            "runner": "nx/tasks-runners/default",
+            "options": {
+                "cacheDirectory": "./.nx-cache",
+                "cacheableOperations": ["prepare", "build", "dev", "test"]
```

Comment:
As I mentioned elsewhere, I'm hesitant about making `test` a cacheable operation. Also is it necessary to put `prepare` here, since `prepare` will ultimately run `build`?

Reply:
Good call, removed prepare from the `cacheableOptions`